libqtile/bar.py

```diff
@@ -289,6 +326,10 @@ def _remove_crashed_widgets(self):
             index = self.widgets.index(i)
             crash = ConfigErrorWidget(widget=i)
             crash._configure(self.qtile, self)
+            if self.horizontal:
+                crash.offsety = self.border_width[0]
+            else:
+                crash.offsetx = self.border_width[3]
```

Comment:
We mentioned this already in another PR - `ConfigErrorWidget` doesn't support vertical bars, does it? Does that same issue affect this? FWIW I don't think that should hold back any PR as the PRs are still improvements, but I'll bump up the priority of getting vertical text boxes working. My last attempt didn't turn out well...

Reply:
These lines aren't affected by that issue (it's actually going to fail on the `crash._configure` line before this. We can leave the lines in as they'll then be ready for when we have vertical text boxes!

Which branch is your vertical textbox on? I can take a look at some point if you want.